### PR TITLE
Voting dapp UI

### LINKED ISSUE #3: Setup voting application
**Task description**
This task pertains to scaffolding the voting application with wallet connections.

**Sub-tasks**
- [x] Create application
- [x] Setup wallet connection management

### PR DESCRIPTION
**Pull request by soerenbf**
## Purpose

Initial setup of voting application, along with connection management for both browser and wallet connect.

Closes #3 

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: apps/voting/src/shared/constants.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { TESTNET } from '@concordium/wallet-connectors';
+
+/**
+ * The Concordium network used for the application.
+ */
+export const NETWORK = TESTNET; // Pull this from env.
```
Comment (abizjak) [inline apps/voting/src/shared/constants.ts:6]: Is this a TODO? Can you please resolve it in this PR?

**Review by abizjak — APPROVED** [review]
Looks fine, but why is there a Chrome.svg in the repository? That seems like an odd way to indicate a browser wallet.

File: apps/voting/src/shell/App/App.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    }, [_connect, isActive, isConnecting]);
+
+    return (
+        <button onClick={connect} className="clear connect-wallet__button">
+            <img
+                className="connect-wallet__button-icon connect-wallet__button-icon--chrome"
+                src={ChromeIcon}
+                alt="wallet connect icon"
+            />
+            Concordium Browser Wallet
```
[the remark this answers is not in the document]
Comment (soerenbf) [reply apps/voting/src/shell/App/App.tsx:46]: Let's go with that then

**Comment by soerenbf** [comment]
> Looks fine, but why is there a Chrome.svg in the repository? That seems like an odd way to indicate a browser wallet.

Thought was, that as it is a chrome extension it would make sense with a chrome icon.. Maybe a concordium icon would make more sense to allow for other chrome extension wallets 🤷

**Pull request merged**